tests/data/expected/main/avro/py314/official_long_list.py

```
@@ -7,7 +7,7 @@
 
 class LongList(BaseModel):
     value: int
-    next: LongList | None = None
+    next: LongList | None
```

Ruff (0.15.15)

[error] Undefined name `LongList`

(F821)